Tests/ApplicativeRouterTests/SyntaxRouterTests.swift

```diff
@@ -59,32 +74,36 @@ class SyntaxRouterTests: XCTestCase {
     )
   }
 
-  func testPostBodyField() {
-    let route = Routes.postBodyField(email: "hello@pointfree.co")
-    let request = URLRequest(url: URL(string: "signup")!)
-      |> \.httpBody .~ Data("email=hello@pointfree.co".utf8)
-      |> \.httpMethod .~ "POST"
-
-    XCTAssertEqual(route, router.match(request: request))
-    XCTAssertEqual(request, router.request(for: route))
-    XCTAssertEqual("signup", router.templateUrl(for: route)?.absoluteString)
-  }
-
-  func testPostBodyJsonDecodable() {
-    let episode = Episode(
-      title: "Intro to Functions", blurb: "Everything about functions!", length: 300, category: nil
-    )
-    let route = Routes.postBodyJsonDecodable(episode: episode, param: 42)
-    let request = URLRequest(url: URL(string: "episodes/42")!)
-      |> \.httpBody .~ (try? JSONEncoder().encode(episode))
-      |> \.httpMethod .~ "POST"
-
-    XCTAssertEqual(route, router.match(request: request))
-    XCTAssertEqual(request, router.request(for: route))
-  }
+//  func testPostBodyField() {
```

Comment:
I just cant get any of the POST tests to work. getting a crash here https://github.com/apple/swift-corelibs-foundation/blob/master/Foundation/NSCFString.swift#L118 when I run these tests